lib/Dialect/SV/Transforms/SVExtractTestCode.cpp

```diff
@@ -346,29 +347,30 @@ inlineInputOnly(hw::HWModuleOp oldMod, hw::InstanceGraph &instanceGraph,
   // declaration with an inner symbol referred by non-bind ops (e.g. hierpath).
   for (auto port : oldMod.getPorts()) {
     if (port.sym) {
-      // Reject if the inner sym is a per-field symbol, or its inner ref is used
-      // by non-bind op.
-      if (!port.sym.getSymName() || port.sym.size() != 1 ||
-          innerRefUsedByNonBindOp.count(hw::InnerRefAttr::get(
-              oldMod.getModuleNameAttr(), port.sym.getSymName()))) {
-        oldMod.emitWarning() << "module " << oldMod.getModuleName()
-                             << " is an input only module but cannot "
-                                "be inlined because a signal "
-                             << port.name << " is referred by name";
-        return;
+      for (auto property : port.sym) {
+        if (innerRefUsedByNonBindOp.count(property.getName())) {
```

Comment:
Should this still use `InnerRef` so use of one inner sym in a module doesn't make them all count as "used"?
(haven't chased it down)

Reply:
Yeah, this should still be using InnerRefAttrs.  There is another similar thing in this pass, `BindTable`, but it maps `map<ModuleName, map<InnerSymName, BindOp>>` so it should be safe to use the (non-globally unique) inner sym name there.